tsl/src/continuous_aggs/utils.c

```diff
@@ -106,31 +106,31 @@ continuous_agg_validate_query(PG_FUNCTION_ARGS)
 			edata->elevel = WARNING;
 			edata->sqlerrcode = ERRCODE_FEATURE_NOT_SUPPORTED;
 			edata->message = "multiple statements are not supported";
-
-			PG_RETURN_DATUM(create_cagg_validate_query_datum(tupdesc, is_valid_query, edata));
```

Comment:
We shouldn't return from PG_TRY w/o PG_END_TRY, some error handling variables are not reset this way.